src/test/java/ZarrTest.java

```diff
@@ -0,0 +1,132 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.glencoesoftware.bioformats2raw.Converter;
+import loci.common.LogbackTools;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class ZarrTest {
+
+  Path fake;
+
+  @Rule
+  public TemporaryFolder output = new TemporaryFolder();
+
+  /**
+   * Set logging to warn before all methods.
+   */
+  @Before
+  public void setup() throws Exception {
+    LogbackTools.setRootLevel("warn");
+  }
+
+  /**
+   * Run the Converter main method and check for success or failure.
+   *
+   * @param additionalArgs CLI arguments as needed beyond "-o output input"
+   */
+  void assertTool(String...additionalArgs) throws IOException {
+    List<String> args = new ArrayList<String>();
+    for (String arg : additionalArgs) {
+      args.add(arg);
+    }
+    args.add("--use-zarr");
+    args.add(fake.toString());
+    args.add(output.newFolder().toPath().toString());
+    try {
+      Converter.main(args.toArray(new String[]{}));
+    }
+    catch (RuntimeException rt) {
+      throw rt;
+    }
+    catch (Throwable t) {
+      throw new RuntimeException(t);
+    }
+  }
+
+  static Path fake(String...args) {
+    Assert.assertTrue(args.length %2 == 0);
+    Map<String, String> options = new HashMap<String, String>();
+    for (int i = 0; i < args.length; i += 2) {
+      options.put(args[i], args[i+1]);
+    }
+    return fake(options);
+  }
+
+  static Path fake(Map<String, String> options) {
+    return fake(options, null);
+  }
+
+  static Path fake(Map<String, String> options,
+          Map<Integer, Map<String, String>> series)
```

Comment:
Not a blocker for this PR, but I'd guess this logic would be more widely useful and could eventually be moved to somewhere more reusable.

Reply:
Agreed. It comes from my work here: https://github.com/spacetx/spacetx-writer/blob/master/src/test/java/spacetx/tests/Helpers.java

Happy to move it to a base library if you have a suggestion.

cc: @dgault @sbesson